language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -527,93 +553,84 @@ writePackageIdTs dir pkgId =
     T.writeFileUtf8 (dir </> "packageId.ts") $ T.unlines
       ["export default '" <> unPackageId pkgId <> "';"]
 
+-- List of texts to vector of Json strings.
+array :: [T.Text] -> Vec.Vector Json.Value
+array = Vec.fromList . map Json.String
+
+-- List of pairs of texts to a hash map of Json strings.
+fields :: [(T.Text, T.Text)] -> HMS.HashMap T.Text Json.Value
```

Comment:
As mentioned above, this can go away if you switch to a hashmap and call `toJSON` on it.